projects/1.20/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,580 @@
+	"item.gobber2.gobber2_ring_dismissal.line1": "将生物推离玩家",
+
+	"item.gobber2.gobber2_ring_enderchest": "末影箱戒指",
+	"item.gobber2.gobber2_ring_enderchest.line1": "允许玩家访问其末影箱",
+	"item.gobber2.gobber2_ring_enderchest.title": "末影箱正忙",
+
+	"item.gobber2.gobber2_ring_explorer": "开拓者戒指",
+	"item.gobber2.gobber2_ring_explorer.line1": "随机将玩家传送到某个位置",
+	"item.gobber2.gobber2_ring_explorer.line2": "最小/最大搜索范围：%s %s",
+
+	"item.gobber2.gobber2_ring_farmer": "农民戒指",
+	"item.gobber2.gobber2_ring_farmer.line1": "适用于许多农作物、树苗、甘蔗、瓜类、竹子等植物",
+	"item.gobber2.gobber2_ring_farmer.line2": "潜行右键催生花草",
+	"item.gobber2.gobber2_ring_farmer.line3": "范围（方块）：%s",
+
+	"item.gobber2.gobber2_ring_haste": "急迫戒指",
+	"item.gobber2.gobber2_ring_haste.line1": "增加玩家破坏方块的速度",
+	"item.gobber2.gobber2_ring_haste.line2": "急迫能力激活：%s",
+
+	"item.gobber2.gobber2_ring_husbandry": "农牧戒指",
+	"item.gobber2.gobber2_ring_husbandry.line1": "使半径10个方块内的动物交配",
+	"item.gobber2.gobber2_ring_husbandry.line2": "依然遵循正常的繁殖周期",
+
+	"item.gobber2.gobber2_ring_leaping": "跳跃戒指",
+	"item.gobber2.gobber2_ring_leaping.line1": "小幅增强跳跃能力",
+
+	"item.gobber2.gobber2_ring_miner": "矿工戒指",
+	"item.gobber2.gobber2_ring_miner.line1": "破坏玩家周围的大部分原版方块",
+	"item.gobber2.gobber2_ring_miner.line2": "右键破坏方块，无掉落物",
+	"item.gobber2.gobber2_ring_miner.line3": "潜行右键破坏方块，有掉落物",
+	"item.gobber2.gobber2_ring_miner.line4": "右键破坏方块，有掉落物",
+	"item.gobber2.gobber2_ring_miner.line5": "潜行右键破坏方块，无掉落物",
+	"item.gobber2.gobber2_ring_miner.line6": "效果范围：11x5x11",
+	"item.gobber2.gobber2_ring_miner.mode.line1": "方块破坏模式：瞬间",
+	"item.gobber2.gobber2_ring_miner.mode.line2": "方块破坏模式：延迟",
+
+	"item.gobber2.gobber2_ring_phoenix": "凤凰戒指",
+	"item.gobber2.gobber2_ring_phoenix.line1": "使玩家在火焰中存活，并且变得更强大",
+
+	"item.gobber2.gobber2_ring_pyro": "火焰戒指",
+	"item.gobber2.gobber2_ring_pyro.line1": "使敌对生物自发燃烧",
+
+	"item.gobber2.gobber2_ring_repair": "修复戒指",
+	"item.gobber2.gobber2_ring_repair.line1": "缓慢修复玩家物品栏里损坏的物品",
+
+	"item.gobber2.gobber2_ring_return": "回城戒指",
+	"item.gobber2.ring_return.line1": "欢迎回家！！",
+	"item.gobber2.ring_return.line2": "必须先通过在床上睡觉设置重生点",
+	"item.gobber2.ring_return.line3": "长按右键回到床的位置",
+	"item.gobber2.ring_return.line4": "只能在主世界中使用",
+
+	"item.gobber2.gobber2_ring_stealth": "匿踪戒指",
+	"item.gobber2.gobber2_ring_stealth.line1": "使玩家不会被怪物察觉到",
+
+	"item.gobber2.gobber2_ring_sunshine": "阳光戒指",
+	"item.gobber2.gobber2_ring_sunshine.line1": "可以把雨天转换成晴天",
+
+	"item.gobber2.gobber2_ring_swiftness": "迅捷戒指",
+	"item.gobber2.gobber2_ring_swiftness.line1": "使玩家获得迅捷效果",
+	"item.gobber2.gobber2_ring_swiftness.line2": "迅捷能力激活：%s",
+
+	"item.gobber2.gobber2_ring_teleport": "传送戒指",
+	"item.gobber2.ring_teleport.line1": "位置设置成功！",
+	"item.gobber2.ring_teleport.line2": "位置已被设置，请先清除。",
+	"item.gobber2.ring_teleport.line3": "位置已清除！",
+	"item.gobber2.ring_teleport.line4": "欢迎来到主世界",
+	"item.gobber2.ring_teleport.line5": "欢迎来到末地",
+	"item.gobber2.ring_teleport.line6": "欢迎来到下界",
+	"item.gobber2.ring_teleport.line7": "仅支持原版我的世界次元。",
```

```suggestion
	"item.gobber2.ring_teleport.line7": "仅支持原版Minecraft的维度。",
```
原版的固有名词应与Minecraft原版一致，如有特殊用法请指出。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **06.专有名词** 一节。
你也可以通过上方评论中，bot生成的对比（diff）文件或 [Minecraft 译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96) 以获取更多的原版翻译帮助。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

## 已修正
- [f0a2008](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/4198/commits/f0a2008759230cf131ce7aa7c4cc2166e0f3bc5c)